images/linux/toolsets/toolset-1604.json

```diff
@@ -41,9 +41,9 @@
             "arch": "x64",
             "platform" : "linux",
             "versions": [
-                "1.13.*",
-                "1.14.*",
```

Comment:
Hello @yuk7! Thank you for the contribution! 
Could you bring these versions (`1.13` and `1.14`) back please? We would like to keep them on the images for now.

Reply:
Thank you for reviewing.
I fixed and pushed them.

Reply:
Hello @MaksimZhukov, thank you for reviewing.
I fixed and pushed them.